nerfstudio/data/utils/dataloaders.py

```diff
@@ -133,6 +134,7 @@ def __iter__(self):
                 collated_batch = self.cached_collated_batch
             elif self.first_time or (
                 self.num_times_to_repeat_images != -1 and self.num_repeated >= self.num_times_to_repeat_images
+                # if it's the first time, we need to 
```

Comment:
need to ... ?

Reply:
ahh this is just a random comment i wrote for myself, will remove this 😅